data/locales.json

```diff
@@ -27,15 +27,6 @@
         "sitemap_priority": 1,
         "sitemap_changefreq": "daily"
       },
-      {
-        "page": "blog",
```

Comment:
Only note here is…do links still work?  We link to `/blog`, does it still work and not localize?

Reply:
tested a few places and I haven't encountered any instances of localisation - but I can also make a note for Sophie to test that out